packages/flutter_tools/templates/app_shared/linux.tmpl/runner/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
And if we need 3.13 now for some reason, shouldn't this match? It doesn't seem like there's any advantage in having a low requirement for one file since they are only used together.